pyanaconda/modules/storage/partitioning/automatic/utils.py

```diff
@@ -213,6 +216,36 @@ def schedule_implicit_partitions(storage, disks, scheme, encrypted=False, luks_f
     return devs
 
 
+def get_default_partitioning():
+    """Get the default partitioning requests.
+
+    :return: a list of partitioning specs
+    """
+    # Get the platform-specific partitioning.
+    partitioning = list(platform.set_default_partitioning())
+
+    # Get the product-specific partitioning.
+    for attrs in conf.storage.default_partitioning:
+        name = attrs.get("name")
+        swap = name == "swap"
+        spec = PartSpec(
+            mountpoint=name if not swap else None,
+            fstype=None if not swap else "swap",
+            lv=True,
+            thin=not swap,
+            btr=not swap,
+            size=attrs.get("min") or attrs.get("size"),
+            max_size=attrs.get("max"),
```

Comment:
Is it fine to store `None` values here?

Reply:
Yes, it is the default values of the `PartSpec` class.